recipes/rubrix/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "rubrix" %}
+{% set version = "0.7.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rubrix-{{ version }}.tar.gz
+  sha256: 70c1832256e344acf08fd4ccb3d4abb8cee45747ca5a208690abcd080076378d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - python >=3.6
+    - httpx ~=0.15.0
```

Comment:
It would probably be more compatible to use more classic >=,< operators here as old versions of conda do not understand this constraint

Reply:
So, this is not necessary? If you think this will be a better idea, I could update the feedstock later with more classic operators.

Reply:
I have opened an issue to make this suggested change: https://github.com/conda-forge/rubrix-feedstock/issues/1